portfolio/src/main/webapp/script.js

```diff
@@ -40,10 +26,13 @@ async function showServerTime() {
     dateContainer.innerText = textFromResponse;
 }
 
-// async function myRespond() {
-//     const responseFromServer = await fetch('/information');
-//     const textFromResponse = await responseFromServer.text();
+async function myRespond() {
+    const responseFromServer = await fetch('/information');
+    const textFromResponse = await responseFromServer.json();
 
-//     const infoContainer = document.getElementById('info-container');
-//     infoContainer.innerText = textFromResponse;
-// }
\ No newline at end of file
+    const infoContainer = document.getElementById('info-container');
+    const random_ind = Math.floor(Math.random() * Object.keys(textFromResponse).length);
+    const info_ind = Object.keys(textFromResponse);
```

Comment:
Minor: Variable naming could help code be more readable to others who look at it. Since `Object.keys` returns an array containing the names of the properties an object has, it may be clearer if the variable `info_ind` was called something like `info_keys`.

Another minor point: Often times the naming convention for JavaScript variables is to use snakeCase instead of underscore_separated_words. This is, of course, a choice, and there isn't anything wrong with using one convention rather than another. I just wanted to point out most JavaScript code you run into may be using snakeCase.

Reply:
Hi, thank you for all suggestions on my code. I followed your advice and also changed my portfolio a little bit. Should I just push the changes in the same request here to let you review or I should start a new one. Thank you so much!